src/hotspot/share/adlc/output_c.cpp

```diff
@@ -4010,6 +4014,22 @@ void ArchDesc::buildMachNode(FILE *fp_cpp, InstructForm *inst, const char *inden
     fprintf(fp_cpp, " );\n");
     // #####
   }
+  if (inst->_flag != nullptr) {
+    Flag* node = inst->_flag;
+    const char* prefix = "Node::";
```

Comment:
You could potentially make the prefix here `Node::PD::`, then the extra `PD::` could be removed from the .ad file (I don't think it really adds much?).

Reply:
Well I thought that it could also be used to add some non-arch specific flags in the future, and with this the keyword would be more generic and allow for this as well.